mandatory/2-function-creation.js

```diff
@@ -6,7 +6,18 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let number = num;
+  if (typeof number === 'number' && number <= 100 && (number%2 === 0)) {
+    return true;
+  }
+  
+  else return false;
+
+  }
```

Comment:
This is fine. You could remove the if/else and do this in one line.
`return(some expression)`
will return true or false depending on if the statement is true or false.

Reply:
Changed it as return( expression)